build/scripts/script.js

```diff
@@ -439,6 +439,7 @@ function initializeSearchEngine() {
             anchorH3: elt.anchorH3,
           });
           pageIndex.push({
+            file: res[i].file,
```

Comment:
It's already present in `searchIndexLinks` no?

Reply:
Yes but If I don't add it there, there is no way to distinguish if 2 items come from the same page

Reply:
To me the `id` property is supposed to make the link between the two or I miss something?